includes/Admin/Consent.php

```
@@ -0,0 +1,312 @@
+		}
+
+		$this->maybe_set_modal_transient();
+	}
+
+	/**
+	 * Flag the consent modal after our plugin is updated via the updater.
+	 *
+	 * @param mixed $upgrader Instance of the upgrader performing the update.
+	 * @param array $data     Array of bulk item update data.
+	 */
+	public function on_upgrader_process_complete( $upgrader, $data ): void {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$upgrader'. (undefined)

(UnusedFormalParameter)